### ISSUE TITLE
Accept Esc as a valid input for search results

### ISSUE THREAD

**Issue by Bekreth** [gopasspw/gopass#1926]
### Summary
Whenever the fuzzy search for passwords pulls up a list of possible options, currently there is no way to gracefully exist the search menu.  The `Esc` character is read in as "\x1b" and isn't accepted as a valid character and `Ctrl + C` doesn't exit the application, it merely catches the `^C` character and, once inputted, then has an unflattering crash and error message.

I'd like to propose adding in a graceful shutdown; hitting the `Esc` character should immediately close out from the menu without an error message

### Steps To Reproduce
Any fuzzy search that returns multiple results, `gopass someCommonString` and then attempt to gracefully exit with `Esc` or `Ctrl + C`

### Expected behavior
Able to gracefully exit the search

### Environment
- OS: Ubuntu 20.04
- Linux ******** 5.4.0-73-generic #82-Ubuntu SMP Wed Apr 14 17:39:42 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
- gopass Version:
```
gopass 1.10.1-git+HEAD go1.13.15 linux amd64
<root>     - gpg 2.2.19 - git 2.25.1
Available Crypto Backends: age, gpgcli, plain
Available Storage Backends: fs, gitfs, ondisk
```
- Installation method: From Source

**Comment by dominikschulz** [comment]
Sounds good, thanks for the suggestion.

**Comment by AnomalRoil** [comment]
Wasn't that at least partially fixed in 1.12.2?

See :

   - [BUGFIX] Honor Ctrl+C while waiting for user input (#1805
   <https://github.com/gopasspw/gopass/pull/1805>, #1800
   <https://github.com/gopasspw/gopass/issues/1800>)

**Comment by Bekreth** [comment]
Yes it did!  I hadn't installed the latest version yet. Thank you for the redirect.
